.github/dependabot.yml

```diff
@@ -1,74 +1,14 @@
 version: 2
-
 updates:
-  - directory: "sentry-delayed_job"
-    package-ecosystem: "bundler"
+  - package-ecosystem: "bundler"
     schedule:
       interval: "daily"
     allow:
       - dependency-type: "direct"
-    ignore:
-      - dependency-type: "all"
-        update-types: ["version-update:semver-patch"]
-      - patterns:
-        - "**/examples/**"
-
-  - directory: "sentry-opentelemetry"
-    package-ecosystem: "bundler"
-    schedule:
-      interval: "daily"
-    allow:
-      - dependency-type: "direct"
-    ignore:
-      - dependency-type: "all"
-        update-types: ["version-update:semver-patch"]
-      - patterns:
-        - "**/examples/**"
-
-  - directory: "sentry-rails"
-    package-ecosystem: "bundler"
-    schedule:
-      interval: "daily"
-    allow:
-      - dependency-type: "direct"
-    ignore:
-      - dependency-type: "all"
-        update-types: ["version-update:semver-patch"]
-      - patterns:
-        - "**/examples/**"
-
-  - directory: "sentry-ruby"
-    package-ecosystem: "bundler"
-    schedule:
-      interval: "daily"
-    allow:
-      - dependency-type: "direct"
-    ignore:
-      - dependency-type: "all"
-        update-types: ["version-update:semver-patch"]
-      - patterns:
-        - "**/examples/**"
-
-  - directory: "sentry-resque"
-    package-ecosystem: "bundler"
-    schedule:
-      interval: "daily"
-    allow:
-      - dependency-type: "direct"
-    ignore:
-      - dependency-type: "all"
-        update-types: ["version-update:semver-patch"]
-      - patterns:
-        - "**/examples/**"
-
-  - directory: "sentry-sidekiq"
-    package-ecosystem: "bundler"
-    schedule:
-      interval: "daily"
-    allow:
-      - dependency-type: "direct"
-    ignore:
-      - dependency-type: "all"
-        update-types: ["version-update:semver-patch"]
-      - patterns:
-        - "**/examples/**"
+    directories:
+      - "sentry-delayed_job"
+      - "sentry-opentelemetry"
+      - "sentry-rails"
+      - "sentry-ruby"
+      - "sentry-resque"
+      - "sentry-sidekiq"
```

Comment:
what about the ignores?

Reply:
@sl0thentr0py turned out it doesn't work like that :( you can't specify dirs/patterns to ignore

Reply:
then we don't need a config at all, just remove it again

Reply:
@sl0thentr0py no, it will help because it will at least not bother us with npm deps. The config tells dependabot to only consider direct gem dependencies from the provided directories.

Reply:
ah right, thx